diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index aaf7205..7eee9a3 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -10,7 +10,7 @@ add_library(${PROJECT_NAME}_static STATIC ${source_files})
 include_directories("./include")
 
 set_target_properties(${PROJECT_NAME} PROPERTIES PUBLIC_HEADER "${header_files}")
-set_target_properties(${PROJECT_NAME}_static PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
+#set_target_properties(${PROJECT_NAME}_static PROPERTIES OUTPUT_NAME ${PROJECT_NAME})
 
 target_link_libraries(${PROJECT_NAME})
 
@@ -27,5 +27,6 @@ endif()
 install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_static
   ARCHIVE DESTINATION lib
   LIBRARY DESTINATION lib
+  RUNTIME DESTINATION bin
   PUBLIC_HEADER DESTINATION include/${PROJECT_NAME}
   )
